WcaOnRails/app/controllers/regional_organizations_controller.rb

```diff
@@ -38,8 +38,13 @@ def create
     @regional_organization.bylaws.attach(params[:regional_organization][:bylaws])
     @regional_organization.extra_file.attach(params[:regional_organization][:extra_file])
     if @regional_organization.save
-      flash[:success] = "Created new Regional Organization"
-      redirect_to edit_regional_organization_path(@regional_organization)
+      flash[:success] = t('.create_success')
+      RegionalOrganizationsMailer.notify_board_of_new_regional_organization_application(current_user, @regional_organization).deliver_later
+      if current_user.can_manage_regional_organizations?
+        redirect_to edit_regional_organization_path(@regional_organization)
+      else
+        redirect_to root_path
```

Comment:
For redirects it's better to use the absolute `_url`. Using `_path` works, but the HTTP spec requires redirects to complete URL, so it's a good practice to do so explicitly =)

Reply:
Good to know, I'm updating this!